vulkaninfo/vulkaninfo.c

```diff
@@ -4902,13 +4866,8 @@ static void AppGroupDump(const VkPhysicalDeviceGroupProperties *group, const uin
                                               .physicalDeviceCount = group->physicalDeviceCount,
                                               .pPhysicalDevices = group->physicalDevices};
 
-    float queue_priority = 1.0f;
```

Comment:
This was added to fix a validation error.
I can add this back later, but if you'd like you can add this back in for this PR.

Reply:
OK, anything other worth preserving?

Reply:
Lines 4905-4911 should be all you need.  The rest of the code was similar to what you are doing in this commit.